frontend/Search.vue

```diff
@@ -35,102 +35,147 @@
-                    <strong>{{ taxFilter.text }}</strong> in
-                </template>
-                <template v-if="database.length == databases.length">
-                    <strong>all available</strong> databases
+                        <v-divider class="my-2"></v-divider>
+
+                        <v-radio-group v-model="iterativeSearch">
+                            <v-tooltip open-delay="300" top>
+                                <template v-slot:activator="{ on }">
+                                    <label v-on="on">
+                                        Iterative search
+                                        <v-icon color="#FFFFFFB3" style="margin-top:-3px" small v-on="on">{{ $MDI.HelpCircleOutline }}</v-icon>
+                                    </label>
+                                </template>
+                                <span>Improve sensitivity of search by performing an iterative search (--num-iterations 3).</span>
+                            </v-tooltip>
+
+                            <v-radio label="On" :value="true"></v-radio>
+                            <v-radio label="Off" :value="false"></v-radio>
+                        </v-radio-group>
+                    </div>
+                </panel>
+
+
+            </v-flex>
+            <v-flex>
+                <panel>
+                <template slot="content">
+                    <div class="actions" :style="!$vuetify.breakpoint.xsOnly ?'display:flex; align-items: center;' : null">
+                    <v-btn color="primary" :block="$vuetify.breakpoint.xsOnly" x-large v-on:click="search" :disabled="searchDisabled"><v-icon>{{ $MDI.Magnify }}</v-icon>&nbsp;Search</v-btn>
+                    <div :style="!$vuetify.breakpoint.xsOnly ? 'margin-left: 1em;' : 'margin-top: 1em;'">
+                        <span><strong>Summary</strong></span><br>
+                        Search <template v-if="taxFilter">
+                            <strong>{{ taxFilter.text }}</strong> in
+                        </template>
+                        <template v-if="database.length == databases.length">
+                            <strong>all available</strong> databases
+                        </template>
+                        <template v-else>
+                            <strong>{{ database.length }}</strong>
+                            <template v-if="database.length == 1">
+                                database
+                            </template>
+                            <template v-else>
+                                databases
+                            </template>
+                            ({{
+                                databases.filter(db => database.includes(db.path)).map(db => db.name).sort().join(", ")
+                            }})
+                        </template> with {{ $STRINGS.APP_NAME }} in <strong>{{ modes[mode] }}</strong> mode<span v-if="iterativeSearch"><strong> iterative</strong></span>.
```

Comment:
Please add iterative to beginning of sentence

Reply:
why?